src/scripts/Providers/OSUI/SharedProviderResources/Flatpickr/scss/_flatpickr.scss

```diff
@@ -658,3 +659,133 @@ body:has(.has-accessible-features) .osui-monthpicker__dropdown {
 		}
 	}
 }
+
+// OS HighContrast Enabled -------------------------------------------------------
+///
+.os-hight-contrast {
+	.flatpickr-rContainer,
+	.flatpickr-days {
+		padding: 5px;
```

Comment:
5px is not part of the OSUI DS scale, should be 4 or 8

Reply:
Done!